src/main/java/apoc/coll/Coll.java

```diff
@@ -887,4 +887,11 @@ public boolean different(@Name("values") List<Object> values) {
 
         return newList;
     }
+
+    @UserFunction
+    @Description("apoc.coll.fill(item, count) - returns a list with the given count of items")
+    public List<Object> fill(@Name("item") String item, @Name("count") long count) {
+        List<Object> newList = new ArrayList<Object>(Collections.nCopies((int) count, item));
```

Comment:
no need for the new ArrayList() just return the result of nCopies

Reply:
Fixed